zebra-state/src/service/arbitrary.rs

```diff
@@ -60,6 +60,30 @@ impl ValueTree for PreparedChainTree {
 pub struct PreparedChain {
     // the proptests are threaded (not async), so we want to use a threaded mutex here
     chain: std::sync::Mutex<Option<(Network, Arc<SummaryDebug<Vec<PreparedBlock>>>)>>,
+    // the height from which to start the chain. If None, starts at the genesis block
+    start_height: Option<Height>,
+}
+
+impl PreparedChain {
+    /// Create a PreparedChain strategy with Heartwood-onward blocks.
+    #[cfg(test)]
```

Comment:
This module is already test-only:
https://github.com/ZcashFoundation/zebra/blob/f09f2a9022afb19d69a7ab53f846730e99a01eff/zebra-state/src/service.rs#L36-L37

Do we need to exclude this method from the `proptest-impl` feature?
Is there a missing dependency that should be `proptest-impl`?

```suggestion
```

Reply:
That makes sense, but it can be a bit confusing if we ever try to use the method from another crate.

Can we either:
* `[#allow(dead_code)]`, or
* make the function `pub`

Reply:
Got it. I'll change this in the validation PR just to avoid another round of review.